### DISCUSSION TITLE
Recent changes appear to have caused regular out-of-sync issues

### DISCUSSION THREAD

**Discussion by bjranson** [reduxjs/redux-devtools#1758] [General]
I've been using this extension for about 5 years now as a professional ReactJS/Redux dev. It has been working stably and with no real issues in all that time. Until the last few weeks, however.

I have noticed that now the extension is regularly not syncing with changes in redux state. This experience is leading to a lot of bafflement about what is going on with the state not making any sense compared to what I'm looking at in the browser, before I remember, again, "Oh, it's gone out of sync again". Result is a lot of frustration and time wasted.

Has anyone else noticed this?

I am using the latest version of the extension, with the latest version of vanilla Google Chrome.

**Comment by kunjanoscd** [comment]
Also seeing similar issues. In addition, looking to our projects redux state can also show redux state of JIRA stuff that is in another browser tab. Going crazy here. Chrome 129.0.6668.60 (Official Build) (x86_64) + MacOS Ventura 13.7
